docker-for-mac/release-notes.md

```diff
@@ -13,6 +13,45 @@ This page contains information about the new features, improvements, known issue
+  - [Docker Compose 1.27.4](https://github.com/docker/compose/releases/tag/1.27.4)
+  - [Go 1.14.7](https://github.com/golang/go/releases/tag/go1.14.7)
+  - [Alpine 3.12](https://alpinelinux.org/posts/Alpine-3.12.0-released.html)
+  - [Kubernetes 1.18.8](https://github.com/kubernetes/kubernetes/releases/tag/v1.18.8)
+  - [Qemu 4.2.0](https://git.qemu.org/?p=qemu.git;a=tag;h=1e4aa2dad329852aa6c3f59cefd65c2c2ef2062c)
+
+### Bug fixes and minor changes
+  - Docker Desktop on macOS 10.13 is now deprecated.
+  - Removed the legacy Kubernetes context `docker-for-desktop`. The context `docker-desktop` should be used instead. Fixes [docker/for-win#5089](https://github.com/docker/for-win/issues/5089) and [docker/for-mac#4089](https://github.com/docker/for-mac/issues/5089).
+  - Adding the application to the dock and clicking on it will launch the container view if Docker is already running.
+  - Added support for emulating Risc-V via Qemu 4.2.0.
+  - Removed file descriptor limit (`setrlimit`) of `10240`. We now rely on the kernel to impose limits via `kern.maxfiles` and `kern.maxfilesperproc`.
+  - Fixed a Mac CPU usage bug by removing the serial console from `hyperkit`, see [docker/roadmap#12]( https://github.com/docker/roadmap/issues/12#issuecomment-663163280). To open a shell in the VM use either `nc -U ~/Library/Containers/com.docker.docker/Data/debug-shell.sock`.
+  - Copy container logs without ansi colors to clipboard. Fixes [docker/for-mac#4786](https://github.com/docker/for-mac/issues/4786).
+  - Fixed automatic start on log in. See [docker/for-mac#4877] and [docker/for-mac#4890].
+  - Fixed bug where the application won't start if the username is too long.
+  - Fixed a bug where adding directories like `/usr` to the filesharing list prevents Desktop from starting. Fixes [docker/for-mac#4488](https://github.com/docker/for-mac/issues/4488)
+  - Fixed application startup if `hosts` is specified inside the Docker `daemon.json`. See [docker/for-win#6895](https://github.com/docker/for-win/issues/6895#issuecomment-637429117)
+  - Docker Desktop always flushes filesystem caches synchronously on container start. See [docker/for-mac#4943](https://github.com/docker/for-mac/issues/4943).
+
+### Known issues
+  -  There is a known issue when using `docker-compose` with named volumes and gRPC FUSE: second and subsequent calls to `docker-compose up` will fail due to the volume path having the prefix `/host_mnt`. To work around this issue, switch back to `osxfs` in Settings. See [docker/for-mac#4859](https://github.com/docker/for-mac/issues/4859).
+  - There is a known issue when enabling Kubernetes where the settings UI fails to update the Kubernetes state. To work around this issue, close and re-open the window.
+  - There is a rare known issue when switching users where the images view continues to show the repositories of the previous user. To work around this issue, close and re-open the window.
+
```

Comment:
Oh it's not possible to add suggestion to line 58. In the Docker Desktop Community 2.3.0.5 the download link should be updated to

```
> [Download](https://desktop.docker.com/mac/stable/48029/Docker.dmg)
```